arex-instrumentation/dynamic/arex-dynamic-common/src/main/java/io/arex/inst/dynamic/common/DynamicClassExtractor.java

```diff
@@ -376,4 +376,12 @@ private int buildNoArgMethodSignatureHash(boolean isNeedResult) {
         }
         return StringUtil.encodeAndHash(String.format("%s_%s_%s", this.clazzName, this.methodName, null));
     }
+
+    String refactorClassName(String className) {
```

Comment:
1. Suggest renaming to normalizeClassName
2. Use `subString` instead `replaceAll` to improve performance